pkg/cloud/openstack/machine/actuator.go

```diff
@@ -249,7 +248,13 @@ func (oc *OpenstackClient) Delete(cluster *clusterv1.Cluster, machine *clusterv1
 	return nil
 }
 
+// TODO cluster-api is only watching for updates to individual Machines, not MachineSets.. why?
```

Comment:
```suggestion
```
`MachineDeployment`s and `MachineSet`s are handled by `cluster-api`, not the providers. The provider only ever looks at the machines, and `cluster-api` is responsible for actuating the changes from the `MachineDeployment`s and `MachineSet`s to the `Machine`s

Reply:
What I actually meant here is that the core `cluster-api` is not propagating changes to the MachineSet to the Machine level. This is probably the wrong spot for the comment, but it's important to test/get this resolved.

Reply:
Changes should be applied from `MachineSet`->`Machine`. This appears to be broken right now as the most recent controller container version is from September, but we should have a build shortly that will hopefully resolve this